catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -363,6 +367,7 @@ private SolrDocumentList handleSpellcheck(
       boolean userSpellcheckIsOn)
       throws IOException, SolrServerException {
 
+    LOGGER.trace("Begin solr spellcheck: query request {}", request);
```

Comment:
❓ Should this only be logged if spellcheck is happening?  Meaning should this be inside the if statement below?

Reply:
done